python_scripts/metrics_sol_02.py

```diff
@@ -91,3 +91,41 @@
 }
 scores = pd.DataFrame(scores)
 scores
+
+# %% [markdown] tags=["solution"]
+# In the Regression Metrics notebook we mentioned the concept of loss function,
+# which is the metric to be optimized when training a model. In the case of a
+# linear regression, the fitting process consists in minimizing the mean squared
+# error (MSE). Some estimators, such as the `HistGradientBoostingRegressor`, can
+# use different loss functions, to be set using the `loss` hyperparameter.
+#
+# Notice that the evaluation metrics and the loss function are not necessarily
+# the same. Let's see an example:
```

Comment:
I don't know if it would be correct mentioning that we use a loss function as a proxy of the final evaluation metric that we want to optimize?